vms/registration/forms.py

```diff
@@ -5,7 +5,7 @@
 
 class UserForm(forms.ModelForm):
     # password not visible when user types it out
-    password = forms.CharField(widget=forms.PasswordInput())
+    password = forms.CharField(widget=forms.PasswordInput(), min_length=8, max_length=25)
```

Comment:
Why do we have two unrelated changes in one PR ?

Reply:
This is the change which fixes issue #578 . The other one is a merged commit. @tonythomas01